tests/unit/nupic/algorithms/cla_classifier_test.py

```diff
@@ -190,6 +191,13 @@ def testComputeCategory(self):
     self.assertSetEqual(set(result.keys()), set(('actualValues', 1)))
     self.assertEqual(result['actualValues'][4], 'D')
 
+    predictResult = c.compute(recordNum=2, patternNZ=[1, 5, 9],
+                              classification={'bucketIdx': 5,
+                                              'actValue': None},
+                              learn=True, infer=True)
+    for value in predictResult['actualValues']:
+      self.assertIsInstance(value, (types.NoneType, types.StringType))
```

Comment:
You can, and I thought about it and concluded that it didn't exactly capture the intent of the test.  The original failure mode was that we were getting floats and None back instead of strs, so I wanted the assertion to match.